src/sentry/migrations/0020_auto_20191125_1420.py

```diff
@@ -14,7 +14,8 @@ def backfill_group_ids(model):
     for attachment in RangeQuerySetWrapper(query, step=1000):
         event = eventstore.get_event_by_id(attachment.project_id, attachment.event_id)
         if event:
-            attachment.update(group_id=event.group_id)
+            attachment.group_id = event.group_id
+            attachment.save(update_fields=("group_id",))
```

Comment:
Could also do my favorite:

```python
EventAttachment.objects.filter(id=attachment.id).update(group_id=...)
```

I personally think this is more clear. But it ultimately does the same thing.

Reply:
I ❤️ one-liners